### PR TITLE
docs(welcome page): expand docs on welcome page

### PR DESCRIPTION
**Pull request by monicawheeler**
# Description

This PR expands and enhances the documentation sections on the Welcome page, providing more detailed explanations and more comprehensive information for users. The changes aim to smooth the onboarding process and ensure users understand the project's features and functionalities better.

## Type of change

- [x] This change requires a documentation update

# How Has This Been Tested?
N/A

**Test Configuration**:
N/A

# Checklist:
Documentation only updates; the following does not apply.
- [NA] My code follows the style guidelines of this project
- [NA] I have performed a self-review of my code
- [NA] I have commented on my code, particularly in hard-to-understand areas
- [NA] I have made corresponding changes to the documentation
- [NA] My changes generate no new warnings
- [NA] I have added tests that prove my fix is effective or that my feature works
- [NA] New and existing tests pass locally with my changes

### CODE REVIEW

File: libs/core/src/stories/Welcome.stories.mdx [hunk 1]
```diff
@@ -3,14 +3,21 @@ import { Meta, Story } from '@storybook/addon-docs';
...
 
 ## Foundations
-Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.
+Our design system is built upon a foundation of three core visual elements: Typography, Color, and Icons. These fundamental components work harmoniously to create a consistent and cohesive visual language throughout our designs.
 
 ## Components
-Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.
-Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.
+Components are the backbone of our design system, acting as the building blocks for crafting seamless and cohesive user interfaces. Our components are available in TypeScript and React, which maximizes usability and ensure compatibility with different development environments.
```
Comment (monicawheeler) [inline libs/core/src/stories/Welcome.stories.mdx]: I especially would like the sentence, "Our components are available in TypeScript and React..." to be reviewed for completeness!

Comment (ju-Skinner) [reply libs/core/src/stories/Welcome.stories.mdx]: Typescript is the "language" that it is written in. It is not a framework or tooling that is consumed.

Comment (monicawheeler) [reply libs/core/src/stories/Welcome.stories.mdx]: Any suggestions on better wording?

File: libs/core/src/stories/Welcome.stories.mdx [hunk 2]
```diff
@@ -3,14 +3,21 @@ import { Meta, Story } from '@storybook/addon-docs';
...
 
 ## Contributing
-We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
+We firmly believe in fostering an inclusive and collaborative environment where everyone feels empowered to participate in the evolution of our design system. The success and longevity of the system depend on the collective efforts and insights of our diverse team, as well as the users who interact with it daily.
+
+We can continuously refine and expand our design system to meet evolving needs and expectations by encouraging open communication, feedback, and contributions from all stakeholders. This collaborative approach helps identify potential improvements and new ideas and promotes a shared sense of ownership and investment in the system's success.
```
Comment (ju-Skinner) [inline libs/core/src/stories/Welcome.stories.mdx]: I feel like we should remove the word `can` from the statement `We can continuously`. It makes it seem like we optionally do it.

File: libs/core/src/stories/Welcome.stories.mdx [hunk 3]
```diff
@@ -3,14 +3,21 @@ import { Meta, Story } from '@storybook/addon-docs';
...
+
+To facilitate this process, we provide [resources and guidelines](/docs/resources-contributing-to-sage--page) on understanding, using, and contributing to the design system. By offering a clear path for participation, we aim to foster a thriving community of designers, developers, and users who work together to ensure the system remains relevant, effective, and resilient over time.
```
Comment (ju-Skinner) [inline libs/core/src/stories/Welcome.stories.mdx]: Should we rename the title of the linked document? e.g resources-contributing-to-the-design-system

File: libs/core/src/stories/Welcome.stories.mdx (hunk 1 above)
Comment (monicawheeler) [reply libs/core/src/stories/Welcome.stories.mdx]: "...available for web components and React..." ?

**Review by pixelflips — APPROVED** [review]

File: libs/core/src/stories/Welcome.stories.mdx [hunk 4]
```diff
@@ -3,14 +3,21 @@ import { Meta, Story } from '@storybook/addon-docs';
...
+Components are the backbone of our design system, acting as the building blocks for crafting seamless and cohesive user interfaces. Our components are available for web components and React, which maximizes usability and ensure compatibility with different development environments.
+
+Our extensive documentation provides code examples, property options, and best practices for each component, making it easy for designers and developers to understand and implement them effectively. This thorough resource empowers teams to create consistent, high-quality experiences while reducing the learning curve and boosting productivity.
+
+In addition to the technical documentation, we also provide links to Figma design files for each component. These design assets visually represent the components and their various states, allowing designers to incorporate them into their mockups and prototypes easily. By integrating code and design resources, our design system is a comprehensive, one-stop solution for creating outstanding experiences for all Kajabi entrepreneurs.
```
Comment (ju-Skinner) [inline libs/core/src/stories/Welcome.stories.mdx]: Is this something we are planning to do and allow publically? Or will the files be there but only those with internal email addresses can see and open them?

Comment (monicawheeler) [reply libs/core/src/stories/Welcome.stories.mdx]: Great question! We should not publish publically, removing this section.

File: libs/core/src/stories/Welcome.stories.mdx [hunk 5]
```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
...
 
 ## Foundations
-Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.
+Our design system is built upon a foundation of three core visual elements: Typography, Color, and Icons. These fundamental components work harmoniously to create a consistent and cohesive visual language throughout our designs.
 
 ## Components
-Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.
-Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.
+Components are the backbone of our design system, acting as the building blocks for crafting seamless and cohesive user interfaces. Our components are available for web components and React, which maximizes usability and ensure compatibility with different development environments.
```
Comment (goodwinchris) [inline libs/core/src/stories/Welcome.stories.mdx]: > Our components are available for web components and React

Should this be?
"Our components are available as web components and React components"

File: libs/core/src/stories/Welcome.stories.mdx [hunk 6]
```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
...
+
+Our extensive documentation provides code examples, property options, and best practices for each component, making it easy for designers and developers to understand and implement them effectively. This thorough resource empowers teams to create consistent, high-quality experiences while reducing the learning curve and boosting productivity.
```
Comment (goodwinchris) [inline libs/core/src/stories/Welcome.stories.mdx]: > This thorough resource empowers teams to create consistent

Maybe change to?
"This thorough documentation empowers teams to create consistent"

File: libs/core/src/stories/Welcome.stories.mdx [hunk 7]
```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
...
 
 ## Contributing
-We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
+We firmly believe in fostering an inclusive and collaborative environment where everyone feels empowered to participate in the evolution of our design system. The success and longevity of the system depend on the collective efforts and insights of our diverse team, as well as the users who interact with it daily.
```
Comment (goodwinchris) [inline libs/core/src/stories/Welcome.stories.mdx:17]: >depend

Maybe? 
"depends"

File: libs/core/src/stories/Welcome.stories.mdx [hunk 8]
```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
...
+
+We continuously refine and expand our design system to meet evolving needs and expectations by encouraging open communication, feedback, and contributions from all stakeholders. This collaborative approach helps identify potential improvements and new ideas and promotes a shared sense of ownership and investment in the system's success.
```
Comment (goodwinchris) [inline libs/core/src/stories/Welcome.stories.mdx]: >improvements and new ideas and promotes a shared sense

Maybe? 
"improvements and new ideas while promoting a shared sense"

File: libs/core/src/stories/Welcome.stories.mdx [hunk 9]
```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
...
+
+To facilitate this process, we provide [resources and guidelines](/docs/resources-contributing-to-the-design-system--page) on understanding, using, and contributing to the design system. By offering a clear path for participation, we aim to foster a thriving community of designers, developers, and users who work together to ensure the system remains relevant, effective, and resilient over time.
```
Comment (goodwinchris) [inline libs/core/src/stories/Welcome.stories.mdx]: > on understanding, using, and contributing

Maybe?
"for understanding, using, and contributing"

**Review by goodwinchris — CHANGES_REQUESTED** [review]
Left a few small wording changes - your call on taking them or leaving them. Feel free to mark them as resolved regardless once you've viewed them.

File: libs/core/src/stories/Welcome.stories.mdx (hunk 7 above)
Comment (monicawheeler) [reply libs/core/src/stories/Welcome.stories.mdx:17]: Thanks for the feedback! I ran this one through Grammarly, and it seems to agree that "depend" is correct, "the singular verb **depends** does not agree with the plural compound subject."

**Review by anechol — APPROVED** [review]
LGTM

File: libs/core/src/stories/Welcome.stories.mdx (hunk 7 above)
Comment (goodwinchris) [reply libs/core/src/stories/Welcome.stories.mdx:17]: Huh, reads wrong to me but 🤷‍♂️

**Review by goodwinchris — APPROVED** [review]

**Pull request merged**